app/config/colors.js

```diff
@@ -0,0 +1,20 @@
+/*
+ *
+ * This file maintains the color palette used across our application.
+ *
+ * Usage -
+ * 1. Define the color variable in this file. Use a meaningful variable name which should
+ *    help us identify which color and/or where that color is primarily used(yet, generic). Examples - white, border-dark, text-description, button-primary etc
+ * 2. Import this file where the color variable/s is/are to be used using the import statement like
+ *    import colors from '../config/colors';
+ * 3. Use the color as - colors.<name_of_variable_defined_here>
+ *
+ */
+
+export default {
+  white: "#FFFFFF",
+  black: "#000000",
+  primary: "#FFBD14",
+  secondary: "#6397E3",
+  background: "#F5F5F5",
```

Comment:
How about we define color name in this formate: `<AppShorthand>_<Color>_<Shade>` ?
```
- sb_red_100: "..." 
- sb_red_300: "..."
- sb_cyan_600: "..."
- sb_blue_700: "..."
```
- This will help us which color are we using and number will represent lighter or darker shade for this
- Also, I have seen this technique being used across many frameworks and apps. For eg: [MaterialDesign](https://materialui.co/colors/)


Also, do you guys think should we avoid some infamy words? and rename them 
- white ---> `sb_bright: "000"`
- black ---> `sb_dark: "fff"`

Reply:
Yes, I like that. Will refactor it.

Reply:
Done

Reply:
Ty!